proto/test.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package proto;
+
+option go_package = "/proto";
+
+import "google/protobuf/any.proto";
+
+message Txn {
```

Comment:
I think this is not used now.

Reply:
All proto stuff has been removed for now. We can create protobuff things later if need them